app/controllers/users_controller.rb

```diff
@@ -51,6 +51,24 @@ def destroy
     end
   end
 
+  def following
+    @title = t "show_follow.following"
+    @user = load_user
+    @users = @user.following
+    @users_with_paginate = @users.paginate page: params[:page],
+      per_page: Settings.follow_per_page
+    render "show_follow"
+  end
+
+  def followers
+    @title = t "show_follow.followers"
+    @user = load_user
+    @users = @user.followers
+    @users_with_paginate = @users.paginate page: params[:page],
```

Comment:
Trong view có 2 chỗ cần danh sách các `followers` của `user`, 1 cái cần toàn bộ, còn 1 cái cần phải phân trang.